app/src/main/res/layout/activity_splash.xml

```diff
@@ -4,6 +4,63 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".SplashActivity">
+    tools:context=".SplashActivity"
+    android:background="@drawable/orangegradiant"
+    >
 
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center_horizontal"
+        android:orientation="vertical"
```

Comment:
Thank you for contributing. Great work @mantyshah. :+1: Also, `android:orientation` is not required for Relative layout.

Reply:
Sorry forgot to remove android:orientation="vertical".